src/teams.json

```diff
@@ -133,7 +133,15 @@
         "googleapis/gapic-generator",
         "googleapis/nodejs-proto-files",
         "googleapis/proto-plus-python",
-        "googleapis/sample-tester"
+        "googleapis/sample-tester",
```

Comment:
Add `api-common-java`?

Not part of this PR, but consider sorting this list alphabetically.